docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,94 @@
 Absolute Solvation Protocol
 ===========================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The absolute solvation protocol calculates the free energy change 
+associate with transferring a molecule from vacuum into a solvent.
+
+.. note::
+   Currently, water is the only supported solvent, however, more solvents might be possible in the future.
+
+The absolute hydration free energy is calculated through a thermodynamic cycle. 
+In this cycle, the interactions of the molecule are decoupled, meaning turned off, both in the solvent and in the vacuum phases.
+The absolute hydration free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/ahfe_thermocycle.png
+   :scale: 100%
+
+   Thermodynamic cycle for the absolute hydration free energy protocol.
+
+Scientific Details
+------------------
+
+Partial annhilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`.AbsoluteSolvationProtocol` the coulombic interactions of the molecule are fully turned off (annihilated),
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
```

Comment:
We should try to reference the soft core potential (if it's a standard one), alternatively just have the equation here.

https://github.com/choderalab/openmmtools/blob/cbff4c8391f56d62f4bd5f0813916cab4c2eea45/openmmtools/alchemy/alchemy.py#L1384-L1387

This is the equation seems like a variation on the Beutler one when you account for the defaults? What do you think?